app/Http/Controllers/V1/ReservationRecurringController.php

```diff
@@ -96,6 +96,10 @@ protected function storeReservation($request, $recurringId)
             $reservationCreated = $this->listCreatedReservation($request, $date, $reservationCreated, $recurringId);
         }
 
+        usort($reservationCreated, function ($a, $b) {
```

Comment:
Ini tujuannya untuk sorting yah kang @iprasetyaa 😁 bisa digunakan collect dari laravel yah

Reply:
Gini ya kang @ayocodingit ?

<img width="391" alt="Screen Shot 2021-05-19 at 10 47 32" src="https://user-images.githubusercontent.com/21275214/118753663-a4d21e80-b88f-11eb-8f3d-bcb2f0c7ad65.png">

Reply:
https://laravel.com/docs/8.x/collections#method-sortby

Reply:
coba seperti ini kang 
```suggestion
        $collection = collect([5, 3, 1, 2, 4])->sortDesc()->values()->all();
```

Reply:
Case datanya gini kang @ayocodingit 

<img width="1105" alt="Screen Shot 2021-05-19 at 10 40 20" src="https://user-images.githubusercontent.com/21275214/118754389-29716c80-b891-11eb-8848-75e931526be6.png">

Reply:
Sudah di fix kang @ayocodingit  https://github.com/jabardigitalservice/reservasi-backend/pull/171/commits/87e27d8ebdf23cfe0f09dedcc71c026216e2373e